src/provider/model/Model.ts

```diff
@@ -379,6 +379,12 @@ export class Model {
             // There is no `window-closed` equivilant for external connections, so we will do our full clean-up of state here
             if (connection.entityType !== EntityType.WINDOW) {
```

Comment:
Why `connection.entityType !== EntityType.WINDOW` here, but `entityType === EntityType.EXTERNAL_CONNECTION` in `onApiHandlerConnection`

Reply:
And similar in `FinEnvironment`, going with `entityType === EntityType.EXTERNAL_CONNECTION || entityType === EntityType.IFRAME`

Reply:
Standardised to just `EXTERNAL_CONNECTION` using `FinConnection`, with all other types using `FinAppWindow`. May tweak that when we add the `VIEW` type.